README.md

```diff
@@ -1,55 +1,23 @@
-<p><a href="https://circleci.com/gh/elastic/search-ui/tree/master"><img src="https://circleci.com/gh/elastic/search-ui/tree/master.svg?style=svg&circle-token=c637bc2af60035a1f4cb5367071999ced238be76" alt="CircleCI buidl"></a>
-  
-> Libraries for the fast development of modern, engaging search experiences. 🎉
+# [Elastic Search UI](https://www.elastic.co/enterprise-search/search-ui)
```

Comment:
This site is marketing fluff. I recommend pointing this to: https://docs.elastic.co/search-ui

Instead of linking this title, I think it's better to spell it out explicitly somewhere... something that says "What you're looking for is not here, go to the documentation site", like we had before:

```
Check out our [full documentation site](https://docs.elastic.co/search-ui) which contains detailed documentation for setting up and using Search UI in your products.
```